src/index.html

```diff
@@ -7,6 +7,318 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <div class="nav__list">
+          <ul class="nav__ul">
+            <li class="nav__link">
+              <a class="is-active nav__a" href="#">Apple</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Samsung</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Смартфоны и телефоны</a>
+            </li>
+            <li class="nav__link">
+              <a data-qa="nav-hover" class="nav__a">Ноутбуки и компьютеры</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Гаджеты</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Планшеты</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Фото</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__a">Видео</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="card card-wrp--card" data-qa="card">
+        <div class="card__img-wrp">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="Моноблок APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+          >
+        </div>
+        <p class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="card__code">
+          Код товара: 195434
+        </p>
+        <div class="card__stars-body">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card-review">
+            Отзывов: 5
+          </p>
+        </div>
+        <p class="card-price">
+          Цена:
+          <span class="card-price__text card-price__text--bold">69 999 грн</span>
+        </p>
+        <div class="card__btn" data-qa="card-hover">
+          <a class="card__link" href="#">Купить</a>
+        </div>
+      </div>
+      <div class="card"  >
```

Comment:
add spaces between blocks

```suggestion
      </div>
      
      <div class="card">
```